src/main.rs

```diff
@@ -14,13 +14,23 @@ fn main() {
         if args[1].ends_with(".yaiwr") {
             run_from_file(&args[1], calc);
         } else {
-            print_result(eval_line(&args[1], calc))
+            eval_statement(&args[1], calc);
         }
     } else {
         repl(calc);
     }
 }
 
+pub fn run_from_file(file_name: &str, calc: &mut Calc) {
+    let file_path = file_name;
+    match fs::read_to_string(file_name) {
+        Ok(content) => {
+            eval_statement(content.as_str(), calc);
+        }
+        Err(_) => print_result(Err(InterpError::ProgramFileNotFound(file_path.to_string()))),
```

Comment:
Should this error be propagated?

Reply:
Good point. changed :point_right: 49fc3b3ca53ca6b4a83d47425b932608b663874e